src/pr.ts

```diff
@@ -49,9 +49,9 @@ export namespace fs {
         return str;
     }
 
-    export function readFile(filePath: string, encoding: string = "utf8"): Promise<any> {
+    export function readFile(filePath: string, encoding: BufferEncoding = "utf8"): Promise<any> {
```

Comment:
Why this change?

Reply:
This is a change due to the update to node 20, the types changed and so we had to match the type change that node20 had.